packages/block-editor/src/store/private-actions.js

```diff
@@ -359,6 +359,20 @@ export function expandBlock( clientId ) {
 	};
 }
 
+/**
+ * @param {Object} value
+ * @param {string} value.rootClientId   The root client ID to insert at.
+ * @param {number} value.insertionIndex The index to insert at.
+ *
+ * @return {Object} Action object.
+ */
+export function setInserterInsertionPoint( value ) {
+	return {
+		type: 'SET_INSERTER_INSERTION_POINT',
+		value,
+	};
+}
+
```

Comment:
I don't like this naming, but it's at least clear 🤷🏻  I'm concerned that this assumes only one instance of the inserter, while the editor package way passes it in. That said, it's still possible to manually pass the inserter the root block and index and it will be respected.